src/components/BoardWizard.vue

```diff
@@ -0,0 +1,333 @@
+      this.toLoad = false;
+    } else if (this.stage === secondStageNumber && this.toLoad) {
+      this.wipLists.forEach((list) => {
+        elem = document.getElementById(`wip_${list}`);
+        if (elem !== null) elem.parentElement.classList.toggle('checkbox-container-selected');
+      });
+      this.archivedLists.forEach((list) => {
+        elem = document.getElementById(`arc_${list}`);
+        if (elem !== null) elem.parentElement.classList.toggle('checkbox-container-selected')
+      });
+      elem = document.getElementById(`back_${this.backlogList}`);
+      if (elem !== null) elem.parentElement.classList.toggle('checkbox-container-selected');
+      elem = document.getElementById(`end_${this.endList}`);
+      if (elem !== null) elem.parentElement.classList.toggle('checkbox-container-selected');
+      elem = document.getElementById(`prod_${this.productionList}`);
+      if (elem !== null) elem.parentElement.classList.toggle('checkbox-container-selected');
+      this.toLoad = false;
+    }
+  },
+  methods: {
+    selectableLists(usedLists) {
+      const uLists = (usedLists === null || usedLists === undefined) ? [] : usedLists;
+      return this.allLists.filter((list) => this.selectedLists.includes(list.id))
+        .filter((list) => this.backlogList !== list.id)
+        .filter((list) => !this.wipLists.includes(list.id))
+        .filter((list) => this.endList !== list.id)
+        .filter((list) => this.productionList !== list.id)
+        .concat(this.allLists.filter((aList) => uLists.includes(aList.id)))
+    },
+    saveData() {
+      if (this.stage === 0) {
+        this.saveBoard();
+        document.getElementById('back_button').classList.remove('button-disabled');
+        document.getElementById('back_button').disabled = false;
+      } else if (this.stage === 1) {
+        this.saveAllLists();
+      } else {
+        this.saveSpecificLists();
+      }
+    },
+    selectBoard(event) {
+      let element = event.target;
+      if (element.tagName === 'P') element = event.target.parentElement;
+      for (const elem of document.getElementsByClassName('wizard--board-selected')) {
+        elem.classList.toggle('wizard--board-selected');
+      }
+      element.classList.toggle('wizard--board-selected');
+      this.selectedBoard = element.id;
+      document.getElementById('save_button').classList.remove('button-disabled');
+      document.getElementById('save_button').disabled = false;
+    },
+    saveBoard() {
+      save('boards', [this.selectedBoard]);
+      this.getLists(this.selectedBoard);
+      this.selectedLists = get(`lists_${this.selectedBoard}`, []);
+
+      const progressBar = document.getElementById('progress_bar');
+      progressBar.classList.remove('wizard--progress-bar--step-1');
+      progressBar.classList.add('wizard--progress-bar--step-2');
+
+      this.stage++;
+      this.toLoad = true;
+    },
+    leaveWizard() {
+      this.$emit('leaveWizard', true);
+    },
+    stepBack(){
```

Comment:
Method 'stepBack' has too many statements (13). Maximum allowed is 10  max-statements<br>Missing space before opening brace                                     space-before-blocks